plugin/xtrabackup/plugin.go

```diff
@@ -302,15 +320,18 @@ func (p XtraBackupPlugin) Backup(endpoint plugin.ShieldEndpoint) error {
 
 	plugin.DEBUG("Executing: `%s`", cmdString)
 	if err = plugin.ExecWithOptions(opts); err != nil {
+		fmt.Fprintf(os.Stderr,"@R{\u2717 Create backup files}\n")
 		return err
 	}
+	fmt.Fprintf(os.Stderr,"@G{\u2713 Create backup files}\n")
```

Comment:
These log message read better as "Creating backup files failed: <err>" and "Created backup files"